### ISSUE TITLE
What about `facets` and `facets_stats` in the v3 ?

### ISSUE THREAD

**Issue by brunonic** [algolia/search-bundle#211]
When I was using an older version, the raw search return a SearchResult object with `facets` and `facets_stats` values, why they are no more returned in the array in the current version v3 ?
It was usefull to create filters.

**Comment by julienbourdeau** [comment]
Hi @brunonic,

`facets` are not retrieved by default in Algolia but you can pass any searchParameters to the `rawSearch` method. If you want to retrieve all facets, you can do:

```php
$postsRaw = $this->indexManager->rawSearch('', Post::class, 1, null, [
        'facets' => ['*']
]);
```
or select the facets you want

```php
$postsRaw = $this->indexManager->rawSearch('', Post::class, 1, null, [
        'facets' => ['tags', 'year']
]);
```

Please let me know what you think.

**Comment by julienbourdeau** [comment]
You're right, I opened a pull request on our doc to add an example under https://www.algolia.com/doc/api-client/symfony/search/?language=symfony#advanced-search
This should be deployed soon.
Thanks for your feedback!
